src/ingest_validation_tools/table-schemas/assays/czems-v1.yaml

```diff
@@ -0,0 +1,138 @@
+doc_url: https://portal.hubmapconsortium.org/docs/assays/maldi-ims
+fields:
+# include: ../includes/vA/all_assays.yaml
+- name: assay_category
+  constraints:
+    enum:
+      - mass_spectrometry_imaging # TODO: This is an internal ID; would prefer not to change.
+- name: assay_type
+  constraints:
+    enum:
+      - CZE-MS
+- name: analyte_class
+  constraints:
+    enum:
+      - protein # NOTE: "proteins" specified, but singular is used elsewhere.
```

Comment:
> maybe. I added an "s" to protein because tne other analytes in that list of enums were all plural. (edited) 
> but that doesn't matter. So we can keep it singular.